scripts/generateSchemas.js

```diff
@@ -21,131 +21,191 @@ const SCHEMA_FILE_EXTENSION = '.schema.json';
  * Defined options for the Schema Generator Claim/Credentials or Both
  */
 const askOptions = () => {
-  const questions = [
-    {
-      type: 'list',
-      name: 'value',
-      message: 'Which schema do you want to generate?',
-      choices: ['UCAs and Claims', 'Credentials', 'Both'],
-      filter: val => val.toLowerCase(),
-    },
-  ];
-  return inquirer.prompt(questions);
+    const questions = [
+        {
+            type: 'list',
+            name: 'value',
+            message: 'Which schema do you want to generate?',
+            choices: ['UCAs and Claims', 'Credentials', 'Both'],
+            filter: val => val.toLowerCase(),
+        },
+    ];
+    return inquirer.prompt(questions);
 };
 
 /**
  * Generate the JSON from an Claim Identifier than generate it's schema saving it on the file system
  * @returns {Promise<void>}
  */
 const generateUcaSchemas = async () => {
-  claimDefinitions.forEach((definition) => {
-    if (definition.credentialItem) {
-      generateSchemaForDefinition(definition, definition.identifier.replace(/-v.*$/, ''), 'claim');
-    }
-  });
+    claimDefinitions.forEach((definition) => {
+        // if (definition.credentialItem) {
```

Comment:
remove comment

Reply:
He should remove this file, since we don't generate schemas anymore